### PR TITLE
Support multiple config files

### PR DESCRIPTION
**Pull request by frankzhao**
Parallelisation moved to per file rather than per product.

### CODE REVIEW

**Comment by frankzhao** [comment]
Will rebase after #10 and #11 are merged

File: main.py [hunk 1]
```diff
@@ -23,11 +25,16 @@
     parser = argparse.ArgumentParser(description='Sync a 3scale API with OpenAPI mappings.')
     parser.add_argument('--3scale_url', dest='url', required=True, help='URL to the 3scale tenant admin.')
     parser.add_argument('--access_token', dest='token', required=True, help='Access token for the 3scale API.')
-    parser.add_argument('--config', dest='config', required=False, default='config.yml', help='Path to config file.')
+    parser.add_argument('--config', dest='config', required=False, default=[], nargs='?', action='append',
```
Comment (xuanshenbo) [inline main.py:28]: rename to `config_file`?

Comment (frankzhao) [reply main.py:28]: Kept the same to maintain compatibility with current scripts

File: main.py [hunk 2]
```diff
@@ -54,23 +93,15 @@
     config = parse_config(loaded_config)
```
Comment (xuanshenbo) [inline main.py:93]: are these 2 lines still needed?

Comment (frankzhao) [reply main.py]: removed

File: sync.py [hunk 3]
```diff
@@ -57,20 +56,37 @@ def sync_mappings(client: ThreeScaleClient, product: Product, product_config: Pr
     ProxyMapping.list(client, product.id)
 
 
-def sync_config(c: ThreeScaleClient, config: Config, open_api_basedir='.', policies_basedir='.', parallel: int = 1):
+def start_sync_for_one_config(client: ThreeScaleClient, config: Config, args):
+    Config.SSL_VERIFY = not args.ssl_disabled
```
Comment (xuanshenbo) [inline sync.py:60]: is this a mistake?

Comment (frankzhao) [reply sync.py:60]: This is required since class variable changes are not persisted when using a multiprocessing pool. I'll add a comment

**Pull request merged**